file-formats/ntta/type/zif_aff_ntta_v1.intf.abap

```diff
@@ -44,9 +44,9 @@ INTERFACE zif_aff_ntta_v1
       "! <p class="shorttext">General Information</p>
       "! General information
       general_information TYPE ty_general_information,
-      "! <p class="shorttext">Assigned Notetypes</p>
-      "! Notetypes assigned to the SAP Object Node Type
-      assigned_notetypes  TYPE ty_notetypes,
+      "! <p class="shorttext">Note Types</p>
+      "! Note Types assigned to the SAP Object Node Type
+      notetypes  TYPE ty_notetypes,
```

Comment:
```suggestion
      note_types  TYPE ty_notetypes,
```
ABAP is snake_case and gets converted to camelCase in JSON

Reply:
done